### PR TITLE
Static package configuration

### PR DESCRIPTION
**Pull request by markrtuttle**
This pull request makes two significant changes to how cbmc-viewer is packaged:

- It switches from dynamic to static configuration of setuptools (the second commit)
- It switches package building and installation from a manual invocation of setuptools to using the build and pip install packages (the third commit)

These are the preferred implementations described in the packaging overviews:

- [Installing packages](https://packaging.python.org/en/latest/tutorials/installing-packages/)
- [Packaging projects](https://packaging.python.org/tutorials/packaging-projects/)

It also moves the cbmc-viewer module under a src directory to be consistent with these documents (the first commit).

This pull requested is organized into three commits to make it easy to review, but they should be squashed into a single commit before merging.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# See https://packaging.python.org/en/latest/tutorials/packaging-projects
+
+[build-system]
+requires = [
+    "setuptools>=42",
+    "wheel"
+]
+build-backend = "setuptools.build_meta"
```
Comment (feliperodri) [inline pyproject.toml:8]: Add new line at the end.

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# See https://packaging.python.org/en/latest/tutorials/packaging-projects
+
+[metadata]
+name = cbmc-viewer
```
Comment (feliperodri) [inline setup.cfg:4]: Isn't the name just `viewer`?

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+version = 2.10
+author = Mark R. Tuttle
+author_email = mrtuttle@amazon.com
+description = A CBMC viewer
```
Comment (feliperodri) [inline setup.cfg:8]: `Scans the output of CBMC and produces a browsable summary of the results.`

File: setup.cfg [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# See https://packaging.python.org/en/latest/tutorials/packaging-projects
+
```
Comment (feliperodri) [inline setup.cfg:2]: nitpick: I'd remove this comment from here.

**Review by feliperodri — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (markrtuttle) [reply pyproject.toml:8]: Done

File: setup.cfg (hunk 2 above)
Comment (markrtuttle) [reply setup.cfg:4]: No, it is cbmc-viewer.

File: setup.cfg (hunk 4 above)
Comment (markrtuttle) [reply setup.cfg:2]: I want to leave pointers to the documentation, it is the best explanation of what is going on.

File: setup.cfg (hunk 3 above)
Comment (markrtuttle) [reply setup.cfg:8]: Thanks.

**Pull request merged**